paddle/fluid/eager/backward.cc

```diff
@@ -149,16 +335,46 @@ void RunBackward(const std::vector<paddle::experimental::Tensor>& tensors,
   std::unordered_map<GradNodeBase*, int> node_in_degree_map =
       getInDegreeMap(queue);
 
+  // Get input's GradNodes and InputMeta Info
+  std::unordered_map<GradNodeBase*, AutogradMeta* /* InputMeta */>
+      input_target_nodes_inputmeta_map;
+  GetTargetNodesInfo(inputs, &input_target_nodes_inputmeta_map);
+
+  // Get Graph Info Betweent input target gradnode and outputs
+  // Record the depending_nodes and potential_stop_nodes
+  std::unordered_map<GradNodeBase* /* child node */,
+                     GradNodeBase* /* father node */>
+      depending_nodes;
+  std::unordered_set<GradNodeBase*> potential_stop_nodes;
+  std::unordered_set<GradNodeBase*> startup_ops;
+  GetGraphInfoBetweenTargets(queue, &input_target_nodes_inputmeta_map,
```

Comment:
Find minimal subgraph algorithm seems redundant for plain backward run. Can we skip this logic if "inputs" argument is empty?

Reply:
Thx, for plain backward run, find minimal subgraph algorithm will call return; in the beginning.